### ISSUE TITLE
Repeated errors in log when device is not available

### ISSUE THREAD

**Issue by andree182** [JurajNyiri/HomeAssistant-Tapo-Control#173]
### Description

When a camera (C210 in my case) is physically turned off/unavailable, the HA log is filled with python backtraces every 5s (with 1 month old code, it was every minute, but longer log...):

```
2022-04-14 09:30:37 ERROR (MainThread) [custom_components.tapo_control] Unexpected error fetching Tapo resource status data: 'updateEntity'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 187, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 147, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/tapo_control/__init__.py", line 235, in async_update_data
    if hass.data[DOMAIN][entry.entry_id]["updateEntity"]._enabled:
KeyError: 'updateEntity'
2022-04-14 09:30:42 ERROR (MainThread) [custom_components.tapo_control] Unexpected error fetching Tapo resource status data: 'updateEntity'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 187, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 147, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/tapo_control/__init__.py", line 235, in async_update_data
    if hass.data[DOMAIN][entry.entry_id]["updateEntity"]._enabled:
KeyError: 'updateEntity'

```

Currently that comprises 99% of my HA log...

### Reproduction Steps

# Turn off camera.
# Profit

### Expected behavior

Just show the camera as unavailable...

### Device Firmware

Firmware: 1.1.16 Build 220119 Rel.41999n(4555)

### Using stream component

Yes

### Does camera work via official integrations?

Yes

### Camera has all attributes filled out in developer tools

No

### HASS Environment

Docker on RPI

### Search for similar issues

Yes

**Comment by andree182** [comment]
This (rather obvious fix) helps:

```
diff --git a/custom_components/tapo_control/__init__.py b/custom_components/tapo_control/__init__.py
index fa075d7..6b686aa 100644
--- a/custom_components/tapo_control/__init__.py
+++ b/custom_components/tapo_control/__init__.py
@@ -232,7 +232,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
                             and entity._enable_sound_detection
                         ):
                             await entity.startNoiseDetection()
-                if hass.data[DOMAIN][entry.entry_id]["updateEntity"]._enabled:
+                if ('updateEntity' in hass.data[DOMAIN][entry.entry_id]) and hass.data[DOMAIN][entry.entry_id]["updateEntity"]._enabled:
                     hass.data[DOMAIN][entry.entry_id]["updateEntity"].updateCam(camData)
                     hass.data[DOMAIN][entry.entry_id][
                         "updateEntity"
```

**Comment by JurajNyiri** [comment]
Hey! Thank you for the report. Would you mind opening a PR please?

**Comment by JurajNyiri** [comment]
Now fixed in [3.7.0](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/3.7.0).
